src/library/cratefeature.cpp

```diff
@@ -110,6 +124,7 @@ CrateFeature::~CrateFeature() {
 #ifdef __AUTODJCRATES__
     delete m_pAutoDjTrackSource;
 #endif // __AUTODJCRATES__
+    delete m_pCreateCrateFolderAction;
```

Comment:
This and the delete above are redundant because it will be deleted bay the Qt Object tree, since it is parented to this.